apps/admin/backend/src/store.test.ts

```diff
@@ -316,14 +320,26 @@ function expectArrayMatch<T>(a: T[], b: T[]) {
 }
 
 describe('getTabulationGroups', () => {
-  const store = Store.memoryStore();
-  const electionId = store.addElection({
-    electionData: electionPrimaryPrecinctSplitsFixtures.asText(),
-    systemSettingsData: JSON.stringify(DEFAULT_SYSTEM_SETTINGS),
-    electionPackageFileContents: Buffer.of(),
-    electionPackageHash: 'test-election-package-hash',
+  let store: Store;
+  let electionId: string;
+  let election: Election;
+
+  beforeEach(async () => {
+    const electionFileContents =
+      electionPrimaryPrecinctSplitsFixtures.electionPackageExport.asBuffer();
```

Comment:
Are you imagining most tests will use the election package fixture? My guess is that many will just need the `Election` from within it, so it might be nice to have an easy way to access that.